### PR TITLE
:book: Update README with zoom meeting info

### PR DESCRIPTION
**Pull request by leec94**
#### What kind of change does this PR introduce?

- :book: update README with current Zoom info

- [x] PR title follows the guidelines defined in our [pull request documentation](https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process)

#### What is the current behavior?

Has old zoom meeting info, there are more meetings for APAC and EMEA now. 

#### What is the new behavior (if this is a feature change)?**

Updated docs, no function change

#### Which issue(s) this PR fixes
https://github.com/ossf/scorecard/issues/3738

#### Does this PR introduce a user-facing change?

For user-facing changes, please add a concise, human-readable release note to
the `release-note`

(In particular, describe what changes users might need to make in their
application as a result of this pull request.)

```release-note

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -585,9 +585,9 @@ Artifact                      | Link
 ----------------------------- | ----
 Scorecard Dev Forum           | [ossf-scorecard-dev@](https://groups.google.com/g/ossf-scorecard-dev)
 Scorecard Announcements Forum | [ossf-scorecard-announce@](https://groups.google.com/g/ossf-scorecard-announce)
-Community Meeting VC          | [Link to z o o m meeting](https://zoom.us/j/98835923979)
-Community Meeting Calendar    | Biweekly Thursdays, 1:00pm-2:00pm PST <br>[Calendar](https://calendar.google.com/calendar?cid=czYzdm9lZmhwNWk5cGZsdGI1cTY3bmdwZXNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
-Meeting Notes                 | [Notes](https://docs.google.com/document/d/1dB2U7_qZpNW96vtuoG7ShmgKXzIg6R5XT5Tc-0yz6kE/edit#heading=h.4k8ml0qkh7tl)
+Community Meeting VC          | [Link to Zoom meeting](https://zoom-lfx.platform.linuxfoundation.org/meeting/95007214146?password=250040c3-80c0-48c4-80c1-07a373116d54)
```
Comment (spencerschrock) [inline README.md:588]: I think the spacing was intentional in the original PR (whether or not it's effective is another story)
https://github.com/ossf/scorecard/pull/1216#discussion_r743008607

Comment (spencerschrock) [reply README.md:588]: although we might not need this row if the 2 below have it

File: README.md [hunk 2]
```diff
@@ -585,9 +585,9 @@ Artifact                      | Link
...
+Community Meeting Calendar    | **_APAC-friendly_** Biweekly on Thursdays at 1:00-2:00 PM Pacific ([OSSF Public Calendar](https://calendar.google.com/calendar/u/0/r/day/2021/11/1?cid=czYzdm9lZmhwNWk5cGZsdGI1cTY3bmdwZXNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)) <br>Video Call: [LFX Zoom](https://zoom-lfx.platform.linuxfoundation.org/meeting/95007214146?password=250040c3-80c0-48c4-80c1-07a373116d54) <br> **_EMEA-friendly_** Every 4 Mondays at 7:00-8:00 AM Pacific ([OSSF Public Calendar](https://calendar.google.com/calendar/u/0?cid=czYzdm9lZmhwNWk5cGZsdGI1cTY3bmdwZXNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)) <br> Video Call: [LFX Zoom](https://zoom-lfx.platform.linuxfoundation.org/meeting/93377638314?password=d53af562-d908-4100-8ae1-52686756cc5d)
```
Comment (spencerschrock) [inline README.md:589]: with regard to the calendar link, that one requires people to sign in and adds it to their calendar.

should we use the embed link from the [getting involved page](https://openssf.org/getinvolved/) instead?

https://calendar.google.com/calendar/u/0/embed?height=600&wkst=1&bgcolor=%238E24AA&showTitle=1&mode=WEEK&showCalendars=0&showTabs=1&showPrint=0&title=OpenSSF+Community+Calendar&src=czYzdm9lZmhwNWk5cGZsdGI1cTY3bmdwZXNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&color=%238E24AA

Comment (leec94) [reply README.md:589]: ive always had to log in to access the zoom meetings. logging in might prevent the zoom bombing concern from the above comment too. is the embedded link the same as the OSSF public calendar link? let me know what needs to be changed for this, thanks!

Comment (spencerschrock) [reply README.md:589]: > ive always had to log in to access the zoom meetings. logging in might prevent the zoom bombing concern from the above comment too.

Logging in to LF to attend the meeting is required by the LF platform i think. Logging in to Google to view the public calendar shouldn't be necessary in my opinion

> is the embedded link the same as the OSSF public calendar link? let me know what needs to be changed for this, thanks!

No, the links are different. I included an example. Try opening that link in a private/incognito tab (so you're not signed in) vs the link you've included.

File: README.md (hunk 1 above)
Comment (leec94) [reply README.md]: added back the spaces for all instances of "zoom"

File: README.md (hunk 2 above)
Comment (leec94) [reply README.md]: added the embedded calendar link for links to the ossf calendar

File: README.md [hunk 3]
```diff
@@ -600,7 +600,13 @@ To report a security issue, please follow instructions [here](SECURITY.md).
...
 #### Zoom
 
-We meet every other Thursday - 4p ET on this [zoom link](https://zoom.us/j/98835923979?pwd=RG5JZ3czZEtmRDlGdms0ZktmMFQvUT09).
+**_APAC-friendly_** Biweekly on Thursdays at 1:00-2:00 PM Pacific ([OSSF Public Calendar](https://calendar.google.com/calendar/u/0/embed?height=600&wkst=1&bgcolor=%238E24AA&showTitle=1&mode=WEEK&showCalendars=0&showTabs=1&showPrint=0&title=OpenSSF+Community+Calendar&src=czYzdm9lZmhwNWk5cGZsdGI1cTY3bmdwZXNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&color=%238E24AA)) 
+
+Video Call: [LFX z o o m](https://zoom-lfx.platform.linuxfoundation.org/meeting/95007214146?password=250040c3-80c0-48c4-80c1-07a373116d54)   
+
+**_EMEA-friendly_** Every 4 Mondays at 7:00-8:00 AM Pacific ([OSSF Public Calendar](https://calendar.google.com/calendar/u/0?cid=czYzdm9lZmhwNWk5cGZsdGI1cTY3bmdwZXNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ))
+
+Video Call: [LFX z o o m](https://calendar.google.com/calendar/u/0/embed?height=600&wkst=1&bgcolor=%238E24AA&showTitle=1&mode=WEEK&showCalendars=0&showTabs=1&showPrint=0&title=OpenSSF+Community+Calendar&src=czYzdm9lZmhwNWk5cGZsdGI1cTY3bmdwZXNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&color=%238E24AA)
```
Comment (spencerschrock) [inline README.md:609]: The OSSF Public Calendar link for the EMEA meetings appears to be the sign-in required one. And the link for the video call is the embed calendar link. So probably a copy/paste error here?

**Review by spencerschrock — APPROVED** [review]
Thanks. Just one small fix required for a copy/paste issue.

File: README.md (hunk 3 above)
Comment (leec94) [reply README.md]: oops, sorry about that! corrected the links

**Pull request merged**